vampyr/CMakeLists.txt

```diff
@@ -13,9 +13,9 @@ target_link_libraries(_vampyr
     MRCPP::mrcpp
   )
 
-add_subdirectory(bases)
-add_subdirectory(functions)
-add_subdirectory(operators)
+#add_subdirectory(bases)
+#add_subdirectory(functions)
+#add_subdirectory(operators)
```

Comment:
Can these be removed entirely?

Reply:
yes, thanks, remnants from the old version with `cpp` files